lib/puppet/type/iis_application.rb

```diff
@@ -41,11 +41,21 @@
   newproperty(:physicalpath, parent: PuppetX::PuppetLabs::IIS::Property::Path) do
     desc 'The physical path to the application directory. This path must be
           fully qualified.'
-    validate do |value|
-      if value.nil? || value.empty?
+    # validate do |value|
```

Comment:
should this commented out code still be here?

Reply:
Most comments left were from Adrian's original code. Probably not needed anymore